cli/lib/tasks/install.js

```diff
@@ -128,6 +128,31 @@ const downloadAndUnzip = ({ version, installDir, downloadDir }) => {
   return Promise.resolve(tasks.run())
 }
 
+const INSTALL_BINARY_TYPES = {
+  URL: (c) => /^https?:\/\/.*/.test(c),
+  FS: (c) => fs.pathExistsSync(c),
```

Comment:
due to file system on Mac locking up with too many M files, we use gracefulfs to retry file access. Which means that ALL file operations have to be async, and we should never use `...Sync` versions. Could you change this code to check for file existence to be async, please?